index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Styles Conference</title>
```

Comment:
-[**optional**] You should consider giving an appropriate title to your project. Preferably a name that briefly describes your project.